src/dbnode/storage/index/types.go

```diff
@@ -65,6 +65,15 @@ type QueryOptions struct {
 	StartInclusive time.Time
 	EndExclusive   time.Time
 	Limit          int
+
+	// Optional param to filter aggregate values.
+	TermFilter *AggregateValuesMap
```

Comment:
break into separate struct - i.e. AggregateQueryOptions v QueryOptions should be separate